nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -357,6 +357,40 @@ services.xserver.displayManager.defaultSession = "xfce+icewm";
       <link linkend="opt-services.httpd.virtualHosts">services.httpd.virtualHosts.&lt;name&gt;.useACMEHost</link>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     For NixOS configuration options, the <literal>loaOf</literal> type has
+     been deprecated and will be removed in a future release. In nixpkgs,
+     options of this type have be rewritten using <literal>attrsOf</literal>
```

Comment:
The options themselves haven't been rewritten to use `attrsOf` though, just the definitions have been changed from lists to attrs.

Reply:
Right, I was thinking too far ahead. Is this better?